src/widgets/helper/ChannelView.cpp

```diff
@@ -1791,13 +1786,14 @@ void ChannelView::handleMouseClick(QMouseEvent *event,
             }
 
             auto &link = hoveredElement->getLink();
-            if (!getSettings()->linksDoubleClickOnly)
+            if (hoveredElement != nullptr &&
+                !getSettings()->linksDoubleClickOnly)
```

Comment:
I realized and considered this after the initial commit, but I didn't bother because I wanted to keep the same _style_ as I had to null check for the next switch case as well and handle it appropriately.